app/views/api/umgap/casestudy_preconfigured.html.erb

```diff
@@ -0,0 +1,137 @@
+<!--   -->                                 Dload  Upload   Total   Spent    Left  Speed
+<!--   -->100  114M  100  114M    0     0  6477k      0  0:00:18  0:00:18 --:--:-- 6332k
+<!--   -->  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+<!--   -->                                 Dload  Upload   Total   Spent    Left  Speed
+<!--   -->100   12G  100   12G    0     0  5076k      0  0:34:12  0:34:12 --:--:-- 5790k</pre>
+      </div>
+      <dl class='dl-horizontal'>
+        <dt><samp>-c dir</samp></dt><dd>Set a different configuration directory.</dd>
+        <dt><samp>-f dir</samp></dt><dd>Link the location of your FragGeneScan installation.</dd>
+        <dt><samp>-d dir</samp></dt><dd>Set a different location for downloading the database files. This can be on a separate disk.</dd>
+        <dt><samp>-y</samp></dt><dd>Download all files of the latest version without asking for confirmation.</dd>
+      </dl>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Analyse</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          When setup is complete, the analyse script will automatically use the downloaded databases and linked tools.
+          Here we will run 2 preconfigured pipelines on the same input data.
+          The input data consists of a 100 paired-end reads sampled from a dataset
+            generated by Lindgreen <i>et al.</i> (2016) for their metabenchmark.
+          With these paired-end reads in the <code>sample-1.fq</code> and <code>sample-2.fq</code> files,
+            the command is as follows.
+        </p>
+        <pre style="max-height: initial;">
+<!--   --><b>$</b> umgap-analyse.sh -1 sample-1.fq -2 sample-2.fq -t tryptic-precision -o tryptic-precision-output.fa -1 sample-1.fq -2 sample-2.fq -t tryptic-sensitivity -z -o tryptic-sensitivity-output.fa.gz
+<!--   --><b>$</b> head -2 tryptic-precision-output.fa
+<!--   --><b>$</b> zcat tryptic-precision-output.fa.gz | head -2</pre>
+      </div>
+      <dl class='dl-horizontal'>
+        <dt><samp>-c dir</samp></dt><dd>Set a different configuration directory.</dd>
+        <dt><samp>-z</samp></dt><dd>Request compression of the output file.</dd>
+        <dt><samp>-1 file</samp></dt><dd>Single ended (or first pair-ended) FASTA or FASTQ input file, optionally compressed.</dd>
+        <dt><samp>-2 file</samp></dt><dd>Second pair-ended FASTA or FASTQ input file, optionally compressed.</dd>
+        <dt><samp>-t</samp></dt><dd>Type of the analysis (high precision by default).</dd>
+        <dt><samp>-o file</samp></dt><dd>The output file.</dd>
+      </dl>
+    </div>
+
+    <div class="card">
+      <div class="card-title card-title-colored"><h2 class="card-title-text">Visualize</h2></div>
+      <div class="card-supporting-text">
+        <p>
+          After analysis, you probably want to view the results.
```

Comment:
Add more information to this section. What does the default format look like? What is the exact format of this first command? count-taxonid-name?